### PR TITLE
BreadCrums finished

### PR DESCRIPTION
**Pull request by THIRUSELVAN-V**
BreadCrums finished

### CODE REVIEW

File: packages/ui/src/components/componentStyled/breadCrums/index.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+export const BreadCrums = ({
+	text,
+	name,
+}: {
+	text?: string;
+	name?: string;
+}) => {
+	return (
+		<div className='bg-white w-[17.063rem] font-[500] text-base flex items-center justify-center gap-1 text-[#02111A]'>
+			<div className='font-[400] text-sm text-[#555555]'>{text}</div>/
```
Comment (premCrayon) [inline packages/ui/src/components/componentStyled/breadCrums/index.tsx]: convert to link tag when i click the breadcrumb redirect to that page

File: packages/ui/src/components/componentStyled/breadCrums/index.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+'use client';
+export const BreadCrums = ({
+	text,
+	name,
+}: {
+	text?: string;
+	name?: string;
+}) => {
+	return (
+		<div className='bg-white w-[17.063rem] font-[500] text-base flex items-center justify-center gap-1 text-[#02111A]'>
```
Comment (premCrayon) [inline packages/ui/src/components/componentStyled/breadCrums/index.tsx]: use fontsize,fontfamily,color from theme

File: packages/ui/src/components/componentStyled/breadCrums/index.tsx (hunk 1 above)
Comment (THIRUSELVAN-V) [reply packages/ui/src/components/componentStyled/breadCrums/index.tsx]: completed

File: packages/ui/src/components/componentStyled/breadCrums/index.tsx (hunk 2 above)
Comment (THIRUSELVAN-V) [reply packages/ui/src/components/componentStyled/breadCrums/index.tsx]: completed

**Pull request merged**